contracts/p0/Distributor.sol

```diff
@@ -21,7 +21,7 @@ contract DistributorP0 is ComponentP0, IDistributor {
     address public constant FURNACE = address(1);
     address public constant ST_RSR = address(2);
 
-    uint8 public constant MAX_DESTINATIONS_ALLOWED = 100;
+    uint8 public constant MAX_DESTINATIONS_ALLOWED = MAX_DESTINATIONS;
```

Comment:
Do we need MAX_DESTINATIONS_ALLOWED? Seems like both are the same concept and value?